core/jvm/src/test/scala/scalaz/zio/RTSSpec.scala

```diff
@@ -304,7 +311,7 @@ class RTSSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec with AroundTime
     // FIXME: Is this an issue with thread synchronization?
     while (reported eq null) Thread.`yield`()
 
-    reported must_=== List(ExampleError)
+    reported.head.asInstanceOf[TerminatedFiber].defect must_=== ExampleError
```

Comment:
Rather than using a cast, you can just use `must_==`.

Reply:
It was the case initially but the test failed. The problem is that I can't compare `TerminatedFiber` directly because the one generated in the code and the one from the test have a different call stack and are not equal. I can only compare `ExampleError` but I need the cast to access it.